site/content/en/docs/Guides/troubleshooting.md

```diff
@@ -328,3 +328,41 @@ kubectl create clusterrolebinding cluster-admin-binding \
 
 On GKE, `gcloud config get-value accounts` will return a lowercase email address, so if
 you are using a CamelCase email, you may need to type it in manually.
+
+## I'm getting stuck in "Terminating" when I uninstall Agones
+
+If you try to uninstall the `agones-system` namespace before you have removed all of the components in the namespace you may
+end up in a `Terminating` state.
+
+```bash
+kubectl get ns
+NAME              STATUS        AGE                                                                                                                                                    
+agones-system     Terminating   4d
+```
+
+Fixing this up requires us to bypass the finalizer in Kubernetes ([article link](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_3.1.1/troubleshoot/ns_terminating.html)), by manually changing the namespace details:
+
+First get the current state of the namespace:
+```bash
+ kubectl get namespace agones-system -o json >tmp.json
+```
+
+Edit the response `tmp.json` to remove the finalizer data, for example remove the following:
+```json
+      "spec": {
+         "finalizers": 
```

Comment:
This is not a valid JSON excerpt. But I see that this was taken from an article.
```suggestion
         "finalizers": [
             "kubernetes"
         ]
```